recipes/cylc/meta.yaml

```diff
@@ -19,7 +19,7 @@ source:
 build:
   # Skip as Cylc requires Linux
   skip: True  # [not linux]
-  number: 0
+  number: 1
```

Comment:
See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string for more:

>The build number should be incremented for new builds of the same version. The number defaults to 0. The build string cannot contain "-". The string defaults to the default conda build string plus the build number.

Reply:
Good to know.